File: Python/1TwoSum.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        m = dict()
        for i, num in enumerate(nums):
            dif = target - num
            if dif in m:
                return [m[dif], i]
            m[num] = i

File: Swift/1TwoSum.swift
class Solution {
    func twoSum(_ nums: [Int], _ target: Int) -> [Int] {
        var freq = [Int : Int]()
        for (i,num) in nums.enumerated() {
            let diff = target - num
            if let index = freq[diff]{
                return [index, i]
            }
            freq[num] = i
        }
        return []
    }
}
